netutil/netutil_test.go

```diff
@@ -0,0 +1,116 @@
+package netutil
+
+import (
+	"net"
+	"os"
+	"testing"
+
+	"github.com/go-kit/log"
+	"github.com/stretchr/testify/assert"
+)
+
+// Setup required logger and example interface names and addresses
+var (
+	logger        log.Logger = log.NewLogfmtLogger(os.Stdout)
```

Comment:
I would remove the global var and just use `log.NewNopLogger()` whenever you need to pass the logger.

Reply:
fixed in commit a0e79a212113d965ef90a75017145a0a4fa17074